docs/reference/builder.md

```diff
@@ -100,7 +100,7 @@ be UPPERCASE in order to distinguish them from arguments more easily.
 
 Docker runs the instructions in a `Dockerfile` in order. **The
 first instruction must be \`FROM\`** in order to specify the [*Base
-Image*](/terms/image/#base-image) from which you are building.
+Image*](/reference/glossary/#base-image) from which you are building.
```

Comment:
Not your fault, but looking at the definition of "base image" in the glossary, I think that definition doesn't apply here. The  definition of "Parent Image" (in the old "terms" document) is a better match here. *However* most people speak of "base image" when talking about `FROM`.

@moxiegirl halp! any ideas how to solve this?